### DISCUSSION TITLE
中州韵助手重构版

### DISCUSSION THREAD

**Discussion by yanhuacuo** [rime/home#1526] [General]
支持平台：MacOS | Windows | Linux
开源协议：不開源
开发框架：Qt6.6

![2024-04-14_194218](https://github.com/rime/home/assets/3107994/6e48f7fc-98e7-4b18-bd41-9514728ec7f8)

![2024-04-14_194300](https://github.com/rime/home/assets/3107994/a2e6e380-64c0-4f6e-81c3-2f654cf26371)

![2024-04-14_194305](https://github.com/rime/home/assets/3107994/e843574d-5d69-4a76-a2e5-4009c178f741)

![2024-04-14_194310](https://github.com/rime/home/assets/3107994/cd084e47-fa1d-475b-bfda-d3138ed0ed17)

![2024-04-14_194315](https://github.com/rime/home/assets/3107994/294a4b95-d0ad-46e2-a977-9f8d89f6090b)

![2024-04-14_194321](https://github.com/rime/home/assets/3107994/01cf9134-1ca0-4e37-bf37-9f94d19e4ffb)

![2024-04-14_194326](https://github.com/rime/home/assets/3107994/ff7e472d-37f1-4142-9b57-fb7390a80364)

![2024-04-14_194331](https://github.com/rime/home/assets/3107994/d5875afa-6b9b-475e-9da1-12b84d5b23d2)

![2024-04-14_194336](https://github.com/rime/home/assets/3107994/dafba9d5-5a58-4e4d-9509-1afc57365ef5)

![2024-04-14_194402](https://github.com/rime/home/assets/3107994/322e3e2f-ba7c-490b-b13e-584799d5e85c)

![2024-04-14_194411](https://github.com/rime/home/assets/3107994/e63034d6-8e35-4b69-95b8-9b601db61444)

![2024-04-14_194436](https://github.com/rime/home/assets/3107994/03d02128-377a-4528-88e9-8ade5337dfa3)

![2024-04-14_194443](https://github.com/rime/home/assets/3107994/fd78d1da-87d3-4661-8981-48709e9da4f9)

![2024-04-14_194516](https://github.com/rime/home/assets/3107994/23ee45fc-3906-431c-9944-44fdbcaff5d9)

![2024-04-14_194530](https://github.com/rime/home/assets/3107994/e60c3765-e06a-4219-b54e-6d7dd7fc5bda)

![2024-04-14_194540](https://github.com/rime/home/assets/3107994/6fabc71d-7484-43f1-8d5d-84c454e9bfaf)

**Comment by yanhuacuo** [comment]
![2024-04-14_195319](https://github.com/rime/home/assets/3107994/5f493336-99d8-4030-9c71-55b33a2ba058)
![2024-04-14_195346](https://github.com/rime/home/assets/3107994/aedfe8a3-a654-40ad-8c15-ad9c2fe4a203)

**Comment by yanhuacuo** [comment]
最近更新：添加主题即时绘制与预览

![2024-04-21_101618](https://github.com/rime/home/assets/3107994/28c9aff4-60e2-4fe3-a4b2-ffc6fee6de53)

![2024-04-21_101635](https://github.com/rime/home/assets/3107994/b7a8b9e1-9639-413e-99d4-595bf2eeda6a)

**Comment by Youxikong** [comment]
那么，在哪里可以买到呢？

**Comment by yanhuacuo** [comment]
https://github.com/rime/home/assets/3107994/525be106-eb5f-4c10-a4b7-9a977875cd5f
